client/blocks/app-banner/index.jsx

```diff
@@ -190,9 +238,31 @@ export class AppBanner extends Component {
 				</Card>
 			</div>
 		);
+
+		const displayJetpackAppBranding = config.isEnabled( 'jetpack/app-branding' );
+
+		return displayJetpackAppBranding ? jetpackAppBanner : wordpressAppBanner;
 	}
 }
 
+export function BannerIcon( props ) {
+	const { translate, currentSection } = props;
+	const isRTL = useRtl();
+	const { icon } = getAppBannerData( translate, currentSection, isRTL );
```

Comment:
Since we already have the data available in the parent component (line 158), what if we just passed it in as props?

Reply:
I've gone ahead to update this as per your suggestion, so that only the `icon` prop is being passed down:

https://github.com/Automattic/wp-calypso/blob/e385f512e946afcf17c16034ebdcdfb4e690f0ac/client/blocks/app-banner/index.jsx#L255-L267

Let me know if anything else is needed here!

Reply:
Nice! LGTM